modules/serverless-build-func-kn.adoc

```diff
@@ -1,3 +1,8 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-getting-started.adoc
+
+:_content-type: REFERENCE
```

Comment:
That's because this was written as a reference module rather than a procedure module. Only procedure modules, i.e. those with numbered steps etc, have prereqs in the modular docs guidelines. If you think this needs to be revised as a procedure module please open a Jira issue for it so that we can follow up, since this PR is just to clean up standard adding prereqs and context attributes, not restructuring whole modules or assemblies.

Reply:
I am not sure what I think, to be honest. Thanks for the clarifying that distinction between procedure and reference modules. I was not aware. I just noticed it when reading through the preview here: https://deploy-preview-41939--osdocs.netlify.app/openshift-enterprise/latest/serverless/functions/serverless-functions-getting-started.html It just seemed odd to me that the document has prerequisites for some things and not for others. It mixes these two kinds of modules together, when there doesn't seem to be a clear reason why. Why would one choose to create one or the other? 

> please open a Jira issue for it so that we can follow up, since this PR is just to clean up standard adding prereqs and context attributes, not restructuring whole modules or assemblies.

I'm happy to if you think it's needed - was just giving feedback on what I was noticing in the PR.